corehq/apps/style/static/style/less/bootstrap3/forms.less

```diff
@@ -94,6 +94,11 @@ legend .subtext {
   }
 }
 
+// Static width for select2 widgets, which otherwise grow too large on case management pages
+.form-control.select2-container {
```

Comment:
Ah, for some reason I went to this and saw the outdated diff still. All good :+1: